boards/beagle/beaglev_fire/doc/index.rst

```diff
@@ -15,11 +15,17 @@ hobbyists, and researchers to explore and experiment with RISC-V technology.
 Building
 ========
 
+There are three board configurations provided for the BeagleV-Fire:
+
+* ``beaglev_fire_polarfire_e51``: Uses only the E51 core
+* ``beaglev_fire_polarfire_u54``: Uses the U54 cores
+* ``beaglev_fire_polarfire_u54_smp``: Uses the U54 cores with CONFIG_SMP=y
+
 Applications for the ``beaglev_fire`` board configuration can be built as usual:
 
 .. zephyr-app-commands::
    :zephyr-app: samples/hello_world
-   :board: beaglev_fire
+   :board: beaglev_fire_polarfire_u54
```

Comment:
The board name should be `beaglev_fire/polarfire/u54`

Reply:
Yes! Good catch 🙏

Reply:
Fixed